_posts/2017-04-17-wego.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Wego
+sourceURL: http://saudigazette.com.sa/business/3-seconds-better-12/
+image: wego.png
+tags:
+  - App Shell
+  - Home Screen
+  - IndexedDB
+  - Offline
+  - Service Worker
+  - Travel
+---
+
+**Wego** reduced page load time from **12** seconds to between **.5** to **three** seconds.
```

Comment:
Should be "**0.5** to **3** seconds" for readability (consistency of number formatting, leading zero so it doesn't look like "5" to speed-readers).

Alternatively **0.5** could also be replaced by the fractional 1/2 character (½).